schema.ts

```diff
@@ -59,6 +58,13 @@ export const moduleSchema = z
       toggle: z.boolean(),
       hatter: z.boolean(),
     }),
+    tags: z.array(
+      z.object({
+        label: z.string(),
+        description: z.string(),
+        value: z.string(),
```

Comment:
@gershido what were the considerations that went into making this value a string rather than a bool?

Reply:
The 'value' property is the tag itself, usually a lower-case string with no spaces. I guess you thought of a bool value in the case where the tag exists on all modules, and the bool activates it or not. But rather, a module will include in its `tags` array only its active tags.

Reply:
ah, I see. makes sense

Reply:
Since tags can't quickly get messy/inconsistent, I wonder if it's better for `value` to be an enum. From my understanding, it looks like a module creator can add whatever tag they want. 

I understand the original purpose of the tag is a catch-all for arbtirary information, but since tags are such an important UX feature, enforcing some level of consistency seems useful. In my opinion, arbtirary information is better left for a text search in the description rather than a tag.

So the schema I'm proposing would be something like 
```
 z.object({
        description: z.string(),
        value: z.nativeEnum(ModuleTagsEnum),
})
```

And the label would be a lookup